nextflow.config

```diff
@@ -47,7 +47,7 @@ params {
 
     // development
     viphog_version = 'v3'
-    meta_version = 'v1'
+    meta_version = 'v2'
```

Comment:
@mberacochea does the CWL pipeline now also use the v2 of the metadata file as the default? I think with the changes to the voting system we introduced it is fine now to switch to v2 as the default. v1 we might only need to reproduce some previous results

Reply:
I was using v1, I'll change to v2 now :+1:

Reply:
alright! and good, then we have the same status for nf and cwl. the v2 has updated viral taxonomy names